.travis.yml

```diff
@@ -33,14 +33,12 @@ matrix:
     # See travisCiBuild task for mandatory combinations
     - jdk: oraclejdk8
       env: VARIANT=2.3
```

Comment:
I suggest that we also remove the exclussion of JDK8/groovy2.3 combination so it will be included by the CI build and of course also remove the comments above.

Travis will just queue the builds.